src/main/kotlin/com/exactpro/th2/conn/amqp/BoxMain.kt

```diff
@@ -101,15 +100,15 @@ fun main(args: Array<String>) {
         )
         resources += service
 
-        parsedRouter.subscribeAll { _, parsedBatch ->
-            parsedBatch.messagesList.forEach { msg ->
+        rawRouter.subscribeAll { _, rawBatch ->
+            rawBatch.messagesList.forEach { msg ->
                 runCatching { service.send(msg) }
```

Comment:
Conn should execute several actions after sending
Success:
- send the raw message with filled metadata to pin with "second", "raw", "publish" attributes (implemented)
- send the event (attach sent message) with information about success sent. Use parent event id from the incoming raw message if exist otherwise conn root event id
Failure:
- send the event with information about failure sent. (implemented)